src/contexts/W3iContext/index.tsx

```diff
@@ -41,28 +31,25 @@ const W3iContextProvider: React.FC<W3iContextProviderProps> = ({ children }) =>
   return (
     <W3iContext.Provider
       value={{
-        chatClientProxy: chatClient,
-        chatProvider,
+        chatClientProxy: null,
         pushProvider,
         authProvider,
         userPubkey,
-        uiEnabled,
+        uiEnabled: { chat: false, ...uiEnabled },
```

Comment:
If `uiEnabled` still contains `chat: true` for some reason it will override the explicit setting based on the way the rest operator behaves if it comes after the explicit setting:

```suggestion
        uiEnabled: { ...uiEnabled, chat: false },
```